_pages/home.md

```diff
@@ -38,7 +38,7 @@ monthly-update:
   - excerpt: "Monthly Community Update - April 2018 [Read Now](/2018/05/05/monthly-update-apr-2018/){: .btn}"
 ---
 
-{% include feature_row id="intro" type="center" %}
+{% include social.html %}
```

Comment:
Any reason why we stop using feature_row but use hard coded social? The former looks simper to me.